controllers/controllers.go

```diff
@@ -132,6 +136,90 @@ func (r *instanceReconciler) instanceFromNode(node *core.Node) (*instance.Info,
 	return instance.NewInfo(addr, username, "", false, node)
 }
 
+// reconcileKubeletClientCA reconcile the CA certificate across all Windows nodes required by kubelet to recognize
+// the kube-apiserver client. No drain or restart required.
+func (r *instanceReconciler) reconcileKubeletClientCA(ctx context.Context, initialCAConfigMap,
+	bundleCAConfigMap *core.ConfigMap) error {
+	r.log.Info("reconciling kubelet CA client certificates with", "ConfigMap", bundleCAConfigMap.Name)
+	// extract CAs form initial CA ConfigMap
+	initialKubeAPIServerServingCABytes, err := certificates.GetCAsFromConfigMap(initialCAConfigMap,
+		certificates.CABundleKey)
+	if err != nil {
+		return err
+	}
+	// extract CAs from new bundle CA ConfigMap
+	kubeAPIServerServingCABytes, err := certificates.GetCAsFromConfigMap(bundleCAConfigMap, certificates.CABundleKey)
```

Comment:
when you name the previous one initialKubeAPI.., you can name this new or current-kubeAPIser...

Reply:
No prefix means current.